popup.js

```diff
@@ -1,9 +1,57 @@
 function getHighlightedText(callback) {
-  chrome.tabs.executeScript( {
-    code: "window.getSelection().toString();"
+  // chrome.tabs.executeScript({
```

Comment:
Is all this commented out code just and old implementation? Can we get delete it or is there still value in keeping it around?